python/spglib/spglib.py

```diff
@@ -116,6 +116,7 @@ def __getitem__(self, key: str) -> Any:
             f"dict interface ({self.__class__.__name__}['{key}']) is deprecated."
             f"Use attribute interface ({self.__class__.__name__}.{key}) instead",
```

Comment:
@ktns do we need to drop this expansion?

Reply:
I have no strong intention to drop that since we will be able to filter it via module.

Reply:
But I thought you found that if the message differs the filter doesn't work.

Reply:
I said that if messages differ they will independently filtered and it cannot be warned only once. I don't want warnings to show up __at all__ during pytest if they are called in a 3rd party module, so I don't care if there's a way to filter them based on caller module.